app/models/candidate_interface/pick_course_form.rb

```diff
@@ -42,7 +42,7 @@ def user_cant_apply_to_same_course_twice
       return if course_id.blank?
 
       if application_form.application_choices.any? { |application_choice| application_choice.course == course }
-        errors[:base] << 'You have already selected this course'
+        errors[:base] << "You have already selected #{course.name_and_code}"
```

Comment:
```suggestion
        errors[:base] << "You have already added #{course.name_and_code} to your application"
```

Reply:
👍

Reply:
Did you intentionally miss out the " to your application" bit?